packages/components/psammead-brand/src/index.jsx

```diff
@@ -73,24 +74,40 @@ const BrandSvg = styled.svg`
   }
 `;
 
-const Brand = ({ brandName, svg }) => (
+/* eslint-disable react/prop-types */
+const StyledBrand = ({ svg, brandName }) => (
+  <Fragment>
+    {svg && (
+      <StyledSpan>
+        <BrandSvg
+          viewBox={`0 0 ${svg.viewbox.width} ${svg.viewbox.height}`}
+          xmlns="http://www.w3.org/2000/svg"
+          focusable="false"
+          aria-hidden="true"
+          ratio={svg.ratio}
+        >
+          {svg.group}
+        </BrandSvg>
+        <VisuallyHiddenText>{brandName}</VisuallyHiddenText>
+      </StyledSpan>
+    )}
+  </Fragment>
+);
+
+/* eslint-disable react/prop-types */
+const BrandWithLink = ({ url, brandName, svg }) => (
+  <StyledLink href={url}>
+    <StyledBrand svg={svg} brandName={brandName} />
+  </StyledLink>
+);
+
+const Brand = ({ brandName, svg, url }) => (
   <StyledWrapper>
-    <ConstraintWrapper>
-      {svg && (
-        <StyledLink href="https://www.bbc.co.uk/news">
-          <StyledSpan>
-            <BrandSvg
-              viewBox={`0 0 ${svg.viewbox.width} ${svg.viewbox.height}`}
-              xmlns="http://www.w3.org/2000/svg"
-              focusable="false"
-              aria-hidden="true"
-              ratio={svg.ratio}
-            >
-              {svg.group}
-            </BrandSvg>
-            <VisuallyHiddenText>{brandName}</VisuallyHiddenText>
-          </StyledSpan>
-        </StyledLink>
+    <ConstraintWrapper hasLink={url}>
+      {url ? (
```

Comment:
We shouldn't really have logic expecting `url` to not be there if it's required. We should either make url not required, or expect it to be there.

Reply:
We can write code to be defensive to errors for require props, e.g. if a required prop is missing, dont throw an error, but that should fail safe, not fail to an expected alternative output

Reply:
Good call. Both you and @rhell are right. Since there is an alternative output, it makes sense to make `url` an optional prop.